ktor-client/ktor-client-cio/common/src/io/ktor/client/engine/cio/CIOEngineConfig.kt

```diff
@@ -76,7 +77,17 @@ public class EndpointConfig {
     /**
      * Maximum number of connection attempts.
      */
-    public var connectRetryAttempts: Int = 1
+    @Deprecated("Misleading name.", replaceWith = ReplaceWith("connectAttempts"))
+    public var connectRetryAttempts: Int
+        get() = connectAttempts
+        set(value) {
+            connectAttempts = value
+        }
+
+    /**
```

Comment:
I know, you just copied the message as is. But it would be nice to clarifiy a bit here. It's important to notice, that this is about connections, not requests. So no request retries caused by this option.

Reply:
Updated. It definitely required making a note, because I also didn't understand this before you mentioned =)